src/Apps/Order/Components/__tests__/AddressVerificationFlow.jest.tsx

```diff
@@ -164,7 +186,7 @@ describe("AddressVerificationFlow", () => {
         context_page_owner_id: "example-order-id",
         context_page_owner_type: "orders-shipping",
         flow: "user adding shipping address",
-        option: "review and confirm",
+        option: "review_and_confirm",
```

Comment:
I made this change as a convenience to pass the value directly. We could keep it the other way if we prefer.

Reply:
reverted